addons/fullscreen/addon.json

```diff
@@ -26,6 +26,15 @@
       "id": "hideToolbar",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Show toolbar when hovered",
+      "id": "hoverToolbar",
+      "type": "boolean",
+      "default": false,
```

Comment:
That sounds like a good idea to me. Because `hideToolbar` is disabled by default, `hoverToolbar` won't do anything until `hideToolbar` is enabled, and then anyone who doesn't like it can turn it off.